boot/makebootable_test.go

```diff
@@ -1653,15 +1721,15 @@ func (s *makeBootable20UbootSuite) TestUbootMakeRunnableSystem20RunModeBootSel(c
 	mockSeedUbootBootSel := filepath.Join(boot.InitramfsUbuntuSeedDir, "uboot/ubuntu/boot.sel")
 	err = os.MkdirAll(filepath.Dir(mockSeedUbootBootSel), 0755)
 	c.Assert(err, IsNil)
-	env, err := ubootenv.Create(mockSeedUbootBootSel, 4096)
+	env, err := ubootenv.Create(mockSeedUbootBootSel, 4096, &ubootenv.CreateOptions{HeaderFlagByte: true})
```

Comment:
these could also use nil as it's equivalent to this default afaict

Reply:
Yeah, it definitely could. I was trying to increase readability (per mvo's above comment), but I can change to using `nil` in the `true` case if that is preferred

Reply:
(bike-shed) I would just go with "nil" here if we just use the defaults :) However maybe the defaults shoudl be done in a way that nil is equaivalent to an empty CreateOptions so the "HeaderFlagByte" needs to be negative by default, so "IgnoreHeaderFlagBytes" or something maybe? Hope my rambling makes sense.